### DISCUSSION TITLE
Disable jump directly to file on single search hit?

### DISCUSSION THREAD

**Discussion by perface** [oracle/opengrok#4553] [Q&A] [answered]
Hi!

We recently upgraded from an ancient version and have noticed the following "new" behavior:

When a Full search returns only one hit (only one file contains the search term), you are redirected to that line in the file instead of the normal search results with the list of files containing the hit.

Can this be disabled?

I.e: Even if there is only one hit, show the search results list instead of redirecting to the file.

Thanks!

**Comment by vladak** [comment, marked as answer]
Not possible at the moment - see https://github.com/oracle/opengrok/issues/3582
